bindings/java/src/main/native/Makefile.am

```diff
@@ -50,6 +50,11 @@ $(JUCX_GENERATED_H_FILES): $(STAMP_FILE)
 
 # Return CPU architecure in Java format
 arch = $(if $(filter x86_64,$(shell uname -p)),amd64,$(shell uname -p))
+
+# Set '-aarch64' suffix only if arch is NOT x86_64
+suffix = $(if $(filter-out x86_64,$(shell uname -p)),-aarch64)
```

Comment:
the suffix should be the machine arch, not (necessarily) aarch64

Reply:
done